docs/simple-post-request-without-twig.md

```diff
@@ -0,0 +1,109 @@
+# Simple POST request without twig in Controller
+
+If your app is fully SPA (Single Page Application) with REST API, certainly you won't be using twig template for pages (except email).
```

Comment:
Could you add this as an example on the main readme instead of its own page?

Reply:
Done.